src/middle/ir_writer.rs

```diff
@@ -363,86 +373,59 @@ impl IRWriter {
             let line = match ssa.g[node] {
                 NodeData::Op(ref opcode, vt) => {
                     if let MOpcode::OpConst(_) = *opcode {
+                        radeco_err!("found const");
```

Comment:
why error?

Reply:
There shouldn't be any const nodes in `inorder_walk` since all the const nodes should be in a separate table and not part of any particular block.

Do you think it might be better as `radeco_warn` instead?

Reply:
No, then is ok. Sorry, oversaw that.